addons/hr_recruitment/models/hr_job.py

```diff
@@ -278,14 +295,14 @@ def _alias_get_creation_values(self):
                 'job_id': self.id,
                 'department_id': self.department_id.id,
                 'company_id': self.department_id.company_id.id if self.department_id else self.company_id.id,
-                'user_id': self.user_id.id,
+                'user_id': self.recruiter.user_id.id,
```

Comment:
Isn't this supposed to be 'recruiter' instead of 'user_id'? If I remember correctly these are the default values for applicants that are created through email applications.
```suggestion
                'recruiter': self.recruiter.user_id.id,
```